sdk/cosmos/azure-cosmos-spark_3-1_2-12/pom.xml

```diff
@@ -103,11 +103,6 @@
       <version>3.1.1</version> <!-- {x-version-update;cosmos-spark_3-1_org.apache.spark:spark-hive_2.12;external_dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
Dependency is declared in the parent POM, resulting in the dependency being declared twice, removing this one